### DISCUSSION TITLE
when i try to run npm run dev i get this error  ELIFECYCLE  Command failed with exit code 1.

### DISCUSSION THREAD

**Discussion by Joshxing** [vercel/next.js#41648] [Help]
how to fix this error when i try to run npm run dev i got this error 
![image](https://user-images.githubusercontent.com/53200365/197342379-c4412beb-35b2-482e-b04b-cd90bc2b6c82.png)

**Comment by thiswallz** [comment]
Try to use plain npm  and not pnpm

**Reply by Joshxing** [reply]
I try but i got the same error
